settings.gradle

```diff
@@ -0,0 +1,3 @@
+
+rootProject.name = 'didcomm'
```

Comment:
`didcomm-jvm` would be preferable for consistency with the github repo

Reply:
Yes, that's probably better from the directory consistency point of view. But, as we discussed previously, it would probably be better to call the published artifact just `didcomm` (without jvm postfix), so we should not forget to explicitly set it when publishing to maven central (if I remember correctly, the default artifact ID is the project name).